magda-web-client/src/Components/Dataset/Add/Pages/ReviewPage/AccessAndUse.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { FunctionComponent } from "react";
+
+import { State } from "../../DatasetAddCommon";
+
+import CollapseBox from "./CollapseBox";
+import CollapseItem from "./CollapseItem";
+
+import DescriptionBox from "Components/Common/DescriptionBox";
+import * as codelists from "constants/DatasetConstants";
+import ValidationRequiredLabel from "../../ValidationRequiredLabel";
+import { shouldValidate } from "../../ValidationManager";
+import { config } from "config";
+import { getFormatIcon } from "../../../View/DistributionIcon";
+
+import "./AccessAndUse.scss";
+
+type PropsType = {
+    stateData: State;
+};
+
+const AccessAndUse: FunctionComponent<PropsType> = props => {
+    const {
+        dataset,
+        datasetPublishing,
+        licenseLevel,
+        distributions,
+        informationSecurity
+    } = props.stateData;
+
+    return (
+        <CollapseBox
+            heading="Access and use"
+            stepNum={3}
+            className="access-and-use"
+        >
+            {config.featureFlags.publishToDga ? (
+                <CollapseItem className="row" alwaysShow={true}>
+                    <div className="col-sm-3">
+                        <div className="title-box">
+                            Publish to data.gov.au as open data (*):
+                        </div>
+                    </div>
+                    <div className="col-sm-9 content-box single-line">
+                        {datasetPublishing?.publishAsOpenData?.["dga"]
+                            ? "YES"
+                            : "NO"}
+                    </div>
+                </CollapseItem>
+            ) : null}
+
+            <CollapseItem
+                className="row"
+                alwaysShow={shouldValidate("$.datasetPublishing.level")}
+            >
+                <div className="col-sm-3">
+                    <div className="title-box">
+                        Who can see the dataset once it is published
+                        <ValidationRequiredLabel validationFieldPath="$.datasetPublishing.level" />
+                        ?
+                    </div>
+                </div>
+                <div className="col-sm-9 content-box single-line">
+                    {datasetPublishing?.level
+                        ? codelists.publishingLevel[datasetPublishing.level]
+                        : "N/A"}
```

Comment:
Can we pull this into a constant ( `NO_VALUE_LABEL` ?) so we can change the copy for all of them at once easily if we want to?

Reply:
Updated